components/standings/commons/standings_storage.lua

```diff
@@ -23,6 +23,37 @@ local ALLOWED_SCORE_BOARD_KEYS = {'w', 'd', 'l'}
 local SCOREBOARD_FALLBACK = {w = 0, d = 0, l = 0}
 local DISQUALIFIED = 'dq'
 
+---@class StandingsTableStorage
+---@field standingsindex integer
+---@field title string?
+---@field type 'ffa'|'swiss'|'league'
+---@field entries StandingEntriesStorage[]
+---@field matches string[]
+---@field roundcount integer
+---@field hasdraw boolean
+---@field hasovertime boolean
+---@field haspoints boolean
+---@field finished boolean
+---@field enddate string? #formated as 'Y-m-d' DEPRECATED
```

Comment:
fwiw for ppt import this value is needed (non ffa case of course)

Reply:
Ah to set the lastplayeddate?

Reply:
yeah
wikis that store the groupscore in lastvsdata will use the edate from standings as date for the placement